src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersDraggableGroup.tsx

```diff
@@ -0,0 +1,69 @@
+import styles from '@common/modules/table-tool/components/TableHeadersDraggableGroup.module.scss';
+import TableHeadersGroupControls from '@common/modules/table-tool/components/TableHeadersGroupControls';
+import TableHeadersReadOnlyList from '@common/modules/table-tool/components/TableHeadersReadOnlyList';
+import getTableHeaderGroupId from '@common/modules/table-tool/components/utils/getTableHeaderGroupId';
+import useTableHeadersContext from '@common/modules/table-tool/contexts/TableHeadersContext';
+import classNames from 'classnames';
+import React from 'react';
+import { Draggable } from 'react-beautiful-dnd';
+
+interface Props {
+  index: number;
+  legend: string;
+  name: string;
+  totalItems: number;
+  onMoveGroupToOtherAxis: () => void;
+}
+
+const TableHeadersDraggableGroup = ({
+  index,
+  legend,
+  name,
+  totalItems,
+  onMoveGroupToOtherAxis,
+}: Props) => {
+  const { groupDraggingActive } = useTableHeadersContext();
+  const groupId = getTableHeaderGroupId(legend);
+
+  return (
+    <>
+      <Draggable draggableId={name} index={index}>
+        {(draggableProvided, draggableSnapshot) => (
+          <div
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...draggableProvided.draggableProps}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...draggableProvided.dragHandleProps}
+            className={classNames(styles.group, {
+              [styles.isDragging]: draggableSnapshot.isDragging,
+              [styles.isDraggingActive]:
+                !draggableSnapshot.isDragging && groupDraggingActive,
+              [styles.isDraggedOutside]:
+                draggableSnapshot.isDragging && !draggableSnapshot.draggingOver,
+            })}
+            ref={draggableProvided.innerRef}
+            role="button"
+            tabIndex={0}
```

Comment:
The list of items are there to provide context for what's in the group so it's clearer what you're moving, so I think having it read for screen reader users is ok, especially as by default it's only two items.
I don't think adding aria-label to the Draggable would help as it is makes it so you can't get to the list easily (or possibly at all, I'm not sure if it's just my poor NVDA skills, but I can't get it to read them out).
I'll be interested to see what DAC says about this though.

Reply:
Yep, thought that'd be the case. It's a little bit of a tricky one tbh. I can imagine that if the user expands the list, every time they refocus the group, it could be annoying having them all read out though.

Guess we can just press on for now and just see what happens from the re-test.